src/component/feature/FeatureToggleList/FeatureToggleListItem/FeatureToggleListItem.tsx

```diff
@@ -95,7 +95,15 @@ export const FeatureToggleListItem = memo<IFeatureToggleListItemProps>(
                                 {/* <span className={styles.listItemToggle}></span> */}
                                 <span></span>
                                 <small>
-                                    <TimeAgo date={createdAt} live={false} />
+                                    <ConditionallyRender
+                                        condition={!!createdAt}
+                                        show={
+                                            <TimeAgo
+                                                date={createdAt as Date}
+                                                live={false}
+                                            />
+                                        }
+                                    />
```

Comment:
Should wrap `show` in an fn to make sure it's not rendered without createdAt, e.g.:

```
<ConditionallyRender
    condition={Boolean(createdAt)}
    show={() => <TimeAgo date={createdAt!} live={false} />}
/>
```